### PR TITLE
recipes-multimedia: Upgrade to NXP release 6.6.3_1.0.0

### PR DESCRIPTION
**Pull request by valijec**
Updated:
        - alsa-lib update to 6.6.3_1.0.0 release and skip QA check for 32bit timer
        - gstreamer1.0-plugins-bad_1.22.5.imx update to 6.6.3_1.0.0 release, enable g2d
        - gstreamer1.0-plugins-base_1.22.5.imx update to 6.6.3_1.0.0 release, add opengl dependencies
        - gstreamer1.0-plugins-good_1.22.5.imx update to 6.6.3_1.0.0 release, enable pulseaudio and 32bit timer
        - gstreamer1.0_1.22.5.imx update to 6.6.3_1.0.0 release and add runtime dependency glibc-gconv-iso8859-5
        - imx-gst1.0-plugin update to 6.6.3_1.0.0 release
        - imx-vpuwrap update to 6.6.3_1.0.0 release

Upgraded:
        - imx-codec: 4.8.2 -> 4.8.3
        - imx-dsp-codec-ext: 2.1.6 -> 2.1.7
        - imx-dsp: 2.1.6 -> 2.1.7
        - imx-parser: 4.8.2 -> 4.8.3
        - imx-opencl-converter: 0.1 -> 0.2.0
New feature:
        - pipewire_1.0.0

### CODE REVIEW

**Comment by valijec** [comment]
This pull request is part of https://github.com/Freescale/meta-freescale/issues/1777

**Comment by valijec** [comment]
> Please take a look at the comments below. Also, please improve the commit log by adding a more descriptive changelog.

Thanks @otavio, I edited the PR with the suggestions

File: recipes-multimedia/gstreamer/gstreamer1.0-plugins-base_1.22.5.imx.bb [hunk 1]
```diff
@@ -118,16 +118,29 @@ SRC_URI:remove = " \
...
 PACKAGECONFIG:remove = "${PACKAGECONFIG_REMOVE}"
-PACKAGECONFIG:append:imxgpu2d = " g2d"
+PACKAGECONFIG_REMOVE ?= "jpeg"
+
+PACKAGECONFIG_GL:imxgpu2d:append:mx6-nxp-bsp = " viv-fb"
+PACKAGECONFIG_GL:imxgpu2d:append:mx7-nxp-bsp = " viv-fb"
+
+PACKAGECONFIG:append = " ${PACKAGECONFIG_G2D}"
+PACKAGECONFIG_G2D          ??= ""
+PACKAGECONFIG_G2D:imxgpu2d ??= "g2d"
```
Comment (thochstein) [inline recipes-multimedia/gstreamer/gstreamer1.0-plugins-base_1.22.5.imx.bb:143]: There is a generic bbappend containing some of this logic already. Please drop what is now duplicated.

Comment (valijec) [reply recipes-multimedia/gstreamer/gstreamer1.0-plugins-base_1.22.5.imx.bb:136]: Thank you @thochstein, that's a good observation

**Review by thochstein — APPROVED** [review]

File: recipes-multimedia/pipewire/pipewire_%.bbappend [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+SRC_URI:append:imx-nxp-bsp = " file://0001-launch-allow-pipewire-pulse-can-be-started-by-root.patch"
+
+SYSTEMD_AUTO_ENABLE:imx-nxp-bsp = "disable"
+
+DEPENDS:append:mx95-nxp-bsp = " libdrm"
+
+PACKAGECONFIG:remove:mx95-nxp-bsp = "libcamera"
+PACKAGECONFIG:remove:imx-nxp-bsp = "gstreamer"
+PACKAGECONFIG:class-target:append = " ${@bb.utils.contains('DISTRO_FEATURES', 'bluetooth', 'bluez-lc3', '', d)}"
```
Comment (otavio) [inline recipes-multimedia/pipewire/pipewire_%.bbappend:11]: This must not change the recipe if not using `nxp-bsp`

Comment (valijec) [reply recipes-multimedia/pipewire/pipewire_%.bbappend]: Thank you, missed that :)

**Pull request merged**